sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/SonarQubePythonIndexer.java

```diff
@@ -128,6 +134,7 @@ private void computeGlobalSymbolsUsingCache(SensorContext context) {
     LOG.info("Partially optimized analysis can be performed for {} out of {} files.", partiallySkippableFiles.size(), mainFiles.size() + testFiles.size());
     // Although we need to analyze all impacted files, we only need to recompute global symbols for modified files (no cross-file dependencies in the project symbol table)
     computeGlobalSymbols(impactfulFiles, context);
+    testFiles.forEach(this::writeContentHashToCache);
```

Comment:
why not do it in forEach created before since there are already multiple operations?